### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by francescoli** [anuranbaka/OpenDTAM#22]
Hi, Paul
Is there any video card requirement of this project? 
I'm using a GT620 video card and get a runtime error after successfully compiling. The log is 

text_file_name = ../../Trajectory_30_seconds/scene_049.txt
Opening: ../../Trajectory_30_seconds/scene_049.png
OpenCV Error: Gpu API call (unknown error) in create, file /home/louis/Library/opencv-2.4.9/modules/gpu/src/matrix_operations.cpp, line 255
terminate called after throwing an instance of 'cv::Exception'
  what():  /home/louis/Library/opencv-2.4.9/modules/gpu/src/matrix_operations.cpp:255: error: (-217) unknown error in function create

Any idea to solve this? 
Best regards
Michel

**Comment by francescoli** [comment]
with cuda-gdb, I get the log 

fatal: The cuda driver initialization failed ( error code = 20 )

**Comment by anuranbaka** [comment]
Yes, At the moment it requires cards with compute capability 3.0 or above
because the kernels use warp shuffle intrinsics and bindless textures. I
could write a version without the warp shuffles pretty easily, but I have
been unable to get old style textures to work. If you could figure out how
to use the old textures, I'd be happy to write the shuffle free version. I
should note that without warp shuffles it will probably run at about half
speed, since the data will need to be loaded twice.

**Comment by francescoli** [comment]
Got it. No need to rewrite, and my new video card is on the way. I will have another try then. 
Thank you, Paul
